cairo-hints/Cargo.toml

```diff
@@ -55,8 +55,10 @@ serde_json = { workspace = true }
 thiserror = { workspace = true }
 cairo-lang-hints-test-runner = { workspace = true }
 cairo-lang-test-plugin = { workspace = true }
+reqwest = { workspace = true }
 handlebars = "6.0.0"
 cainome-cairo-serde = { workspace = true }
+tokio = "1.35.1"
```

Comment:
Why adding tokio on this? If its just to make the async call to the server I would recommend to use the [`blocking client`](https://docs.rs/reqwest/latest/reqwest/blocking/index.html) as we dont need concurrency for cairo hints IMO.